packages/flutter/test/cupertino/scrollbar_test.dart

```diff
@@ -42,25 +45,4 @@ void main() {
       color: const Color(0x15777777),
     ));
   });
-
```

Comment:
Grouping all the tests is ok, we should still specifically test a CupertinoScrollbar and make sure it works with other cupertino widgets, such as putting a scaffold with a nav bar and a scrollbar under it.